pkg/performanceprofile/controller/performanceprofile/resources/resources.go

```diff
@@ -1,21 +1,22 @@
-package controller
+package resources
 
 import (
 	"context"
 	"encoding/json"
 	"reflect"
 
-	apiconfigv1 "github.com/openshift/api/config/v1"
-	mcov1 "github.com/openshift/api/machineconfiguration/v1"
-	tunedv1 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/tuned/v1"
-
 	nodev1 "k8s.io/api/node/v1"
 	apiequality "k8s.io/apimachinery/pkg/api/equality"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/klog"
 	kubeletconfigv1beta1 "k8s.io/kubelet/config/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
just a general thought,  can we avoid using controller-runtime in case we would like to eliminate it completely in the future ?

Reply:
Not in the current architecture.
We're using operator SDK for developing PAO and the controller-runtime client is part of the operator SDK framework.